test/blocks/merch-card/merch-card.test.js

```diff
@@ -6,59 +6,72 @@ const { default: init } = await import('../../../libs/blocks/merch-card/merch-ca
 describe('Merch Card', () => {
   it('Shows segment card', async () => {
     document.body.innerHTML = await readFile({ path: './mocks/segment-card.html' });
-    await init(document.querySelector('.merch-card'));
-    const inner = document.querySelector('.consonant-SegmentBlade-inner');
-    const cardFooter = inner.querySelector('.consonant-CardFooter');
-    const buttons = cardFooter.querySelectorAll('.con-button');
-
-    expect(document.querySelector('.consonant-ProductCard')).to.exist;
-    expect(inner.querySelector('.consonant-SegmentBlade-title')).to.exist;
-    expect(inner.querySelector('.consonant-SegmentBlade-description')).to.exist;
-    expect(cardFooter.querySelector('.con-button')).to.exist;
+    await init(document.querySelector('.segment'));
+    const merchCard = document.querySelector('merch-card');
+    const heading = merchCard.querySelector('h3[slot="heading-xs"]');
+    const body = merchCard.querySelector('div[slot="body-xs"]');
+    const footer = merchCard.querySelector('div[slot="footer"]');
+    const buttons = footer.querySelectorAll('.con-button');
+    expect(merchCard).to.exist;
+    expect(body).to.exist;
+    expect(heading).to.exist;
+    expect(merchCard.getAttribute('variant')).to.be.equal('segment');
+    expect(heading.textContent).to.be.equal('Lorem ipsum dolor sit amet');
+    expect(body.textContent).to.be.equal('Maecenas porttitor congue massa. Fusce posuere, magna sed pulvinar ultricies, purus lectus malesuada libero, sit amet commodo magna eros quis urna. Nunc viverra imperdiet enim.See what\'s included | Learn more');
```

Comment:
While we can generate descriptions/text easily with AI, I don't like using long form lorem ipsum texts. But is is trivial.